packages/core/src/components/highlight/highlight.css

```diff
@@ -0,0 +1,63 @@
+.hds-highlight {
+  --accent-line-color: var(--color-bus);
+  --text-color: var(--color-black-90);
+  --width: 536px;
```

Comment:
Could a variable be used? Or some note where the magic number comes from?

Same with "264px" below